app/scripts/modules/ecs/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -153,6 +160,23 @@ export class EcsServerGroupConfigurationService {
       find<IRegion>(command.backingData.credentialsKeyedByAccount[command.credentials].regions, { name: command.region }).availabilityZones;
   }
 
+  public configureAvailableMetricAlarms(command: IEcsServerGroupCommand): void {
+    command.backingData.filtered.metricAlarms = chain(command.backingData.metricAlarms)
+      .filter({ accountName: 'continuous-delivery-ecs' }) // TODO(Bruno Carrier): delete this line and enable the line below once accounts are properly handled after load balancers are resolved
+      .flattenDeep<MetricAlarmDescriptor>()
+      .filter({ region: command.region })
+      .flattenDeep<MetricAlarmDescriptor>()
+      .map(metricAlarm => {
+        return {
+          accountName: metricAlarm.accountName,
+          region: metricAlarm.region,
+          alarmName: metricAlarm.alarmName,
```

Comment:
You don't need accountName and `region` here --> you have already filtered them, so no need to have them appear in the pipeline

Reply:
Done.